fog/view/enclave/impl/src/lib.rs

```diff
@@ -352,10 +360,31 @@ where
         )
     }
 
+    // Takes each MultiViewStoreResponseQuery
```

Comment:
:thought_balloon: I would suggest creating a unit test for this. This is a nice isolated piece of logic.  

The `query_response` might be a bit hefty to create for each test...

Reply:
Do we have unit tests in enclaves?

Reply:
blarg, I thought there is a way if it doesn't depend on the SGX SDK directly, but maybe it's only for support libs the enclaves rely on.
I can understand punting on it for this PR due to the possible complexity there :(.

Reply:
I just double checked with james, and you're right- since it doesn't depend on SGX SDK we can unit test it. I actually change this method slightly in a follow up PR so I'll add it there.